### DISCUSSION TITLE
Swiper not working in multiple pages

### DISCUSSION THREAD

**Discussion by hilout** [nolimits4web/swiper#4952] [Q&A]
Hello,
I'm using the swiper in multiple pages. The first swiper works fine, but I navigate to the second page, the swiper doesn't swipe.

Any idea?

Thank you.

"swiper": "^6.8.4",
"@ionic/angular": "^5.5.2",

**Comment by vltansky** [comment]
Demo? 
Also take a look at: https://ionicframework.com/docs/api/slides

**Reply by hilout** [reply]
Hi,
Here is the demo: https://codesandbox.io/s/swiper-navigation-angular-forked-fp2o8

I think ion-slide will be removed from ionic 7.x
